app/views/option_sets/export.xlsx.axlsx

```diff
@@ -4,7 +4,12 @@ xlsx_package.workbook.tap do |wb|
   header_style = wb.styles.add_style(b: true, alignment: { horizontal: :center })
   row_style = wb.styles.add_style(locked: false)
 
-  locale = @option_set.mission.setting.preferred_locales.first || :en
+  if @option_set.mission
+    locale = @option_set.mission.setting.preferred_locales.first || :en
```

Comment:
Ugh LoD! 

(I know this is not your code).

Could we refactor though by adding a `default_locale` method to Mission that would delegate to a method of the same name in Setting that would do `preferred_locales.first`?